### ISSUE TITLE
Overview for own feedback

### ISSUE THREAD

**Issue by s0564704** [astuto/astuto#429]
Hi @riggraz I really appreciate your work!

Is is possible to implement an overview where you can check the status and comments of feedback you submitted yourself or posts you have commented on?

Basically is list where the user can see his past interactions with the feedback tool

Greetings

**Comment by riggraz** [comment]
Hi @s0564704, thanks for your suggestion!

You mean a list of posts the user has interacted with? E.g. by posting it, commenting it, voting it, etc.? Similar to what you can see in GitHub's home page (see image):

![Screenshot 2024-11-06 at 11 59 28](https://github.com/user-attachments/assets/5d255364-94f3-4c92-b625-6bbaab8aef69)

Do you think it should only be visible to administrators or also users? I personally think the latter may be better.

**Comment by s0564704** [comment]
Yes, just like that, so users can see an overview of their contributions.
Originally I thought about a personal overview only for the users themselfe. 
But now as you have mentioned it, some kind of analytics for administrators or a scoreboard  might be neat too.
People love to be competitive, a scoreboard could encourage them to write good feedback.
